hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/records/SnapshotModeOp.java

```diff
@@ -590,6 +591,8 @@ private void matchSingleValues(
                 Assertions.assertTrue(
                         (long) expected - (long) actual >= 0,
                         "AccountNum '" + expected + "' was not greater than '" + actual + mismatchContext.get());
+            } else if ("name".equals(fieldName) && matchModes.contains(NONDETERMINISTIC_TOKEN_NAMES)) {
+                Assertions.assertTrue(expected != null && actual != null, "Token name is null");
```

Comment:
What about,
```
Assertions.assertEquals(((String) expected).size(), ((String) actual).size())
```
?

Reply:
Yes that makes sense. But it uses a randomNumber, in `salted` method